src/inmanta/env.py

```diff
@@ -472,30 +484,51 @@ def _gen_content_requirements_file(cls, requirements_list: List[str]) -> str:
 
         return requirements_file
 
-    def install_from_list(self, requirements_list: List[str]) -> None:
+    def install_from_list(
+        self,
+        requirements_list: Sequence[str],
+        *,
+        upgrade: bool = False,
+        upgrade_strategy: PipUpgradeStrategy = PipUpgradeStrategy.ONLY_IF_NEEDED,
+    ) -> None:
         """
         Install requirements from a list of requirement strings. This method uses the Python package repositories
         configured on the host.
 
-        This method differs from the `_install_from_index()` method in the sense that it calls
-        `_gen_content_requirements_file()`, which rewrites the requirements from pep440 format to a format that pip understands.
-        This method is maintained for V1 modules only.
```

Comment:
Didn't get rid of this, just moved it to the called method, where I believe it's more appropriate.